build-and-push-docker-image.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
+
+set -e
+set -x
+
+tag="$(git rev-parse head)"
+image="europe-north2-docker.pkg.dev/sokkel-shared/docker/marketing:${tag}"
+
+docker build --platform linux/amd64 -t "${image}" .
```

Comment:
I _think_ the platform matters for nginx. surprisingly(?), it doesn't matter for caddy. still, nginx does fewer things out of the box, so I think I prefer this. (also, it works.)